cmd/alert.go

```diff
@@ -173,7 +173,7 @@ inactive = 0`,
 		}
 
 		counterAlert := counterFiring + counterPending + counterInactive
-		if len(cliAlertConfig.AlertName) > 1 || counterAlert > 1 {
```

Comment:
No that's not it. But it's the condition being wrong, since it does not account for when there are no alerts being returned.

Reply:
I reworked the alert.go a bit. Now it's a bit clearer